container-images/pragmatic/build_pragmatic.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+set -exu -o pipefail
+
+export  PYTHON_VERSION="python3 -m"
+if [ "$(python3 --version)" \< "Python 3.11" ]; then
+    dnf install -y python3.11 python3.11-pip
```

Comment:
Question, python3.12 and python3.13 are not supported? Still learning these bits.

Reply:
They are supported, this is just checking for RHEL 9 systems with python 3.9.  Basically ramalama requires 3.11 or later.